eos/db/migrations/upgrade16.py

```diff
@@ -0,0 +1,14 @@
+"""
+Migration 16
+
+- Alters fits table to introduce notes attribute
+"""
+
+import sqlalchemy
+
+def upgrade(saveddata_engine):
+    # Update fits schema to include notes attribute
+    try:
+        saveddata_engine.execute("SELECT notes FROM fits LIMIT 1")
```

Comment:
I ripped this off from https://github.com/pyfa-org/Pyfa/blob/master/eos/db/migrations/upgrade13.py